GuessTheWord-Starter/app/src/main/java/com/example/android/guesstheword/screens/score/ScoreFragment.kt

```diff
@@ -45,6 +51,15 @@ class ScoreFragment : Fragment() {
                 false
         )
 
+        val args: ScoreFragmentArgs by navArgs()
+        // val args: ScoreFragmentArgs.fromBundle(getArguments()) doesn't work...?
```

Comment:
Is this meant to be left here? If not, please delete this line

Reply:
It was intended. I wanted to remind myself to ask for your input on why it doesn't work. According to the documentation, the commented line is one of the ways to retrieve arguments passed using SafeArgs, but here it wasn't working. I tried cleaning/rebuilding, but to no avail.